.yamato/wrench/wrench_config.json

```diff
@@ -0,0 +1,27 @@
+{
+  "project_path": "Tools/CineMachine-Recipes",
+  "schema_version": 0.5,
+  "packages": {
+    "com.unity.cinemachine": {
+      "directory": "com.unity.cinemachine/",
+      "prePackCommands": [],
+      "preTestCommands": {},
+      "InternalOnly": false,
+      "NeverPublish": false
+    }
+  },
+  "releasing_packages": [
+    "com.unity.cinemachine"
+  ],
+  "jobs_to_monitor": {
+    "com.unity.cinemachine": [
+      ".yamato/wrench/preview-a-p-v.yml#all_preview_apv_jobs",
+      ".yamato/wrench/api-validation-jobs.yml#api_validation_-_cinemachine_-_2022_3_-_windows",
+      ".yamato/wrench/promotion-jobs.yml#publish_dry_run_cinemachine"
+    ]
+  },
+  "publishing_job": ".yamato/wrench/promotion-jobs.yml#publish_cinemachine",
+  "branch_pattern": "ReleaseSlash",
```

Comment:
What does this mean?

Reply:
This config file is read by the backend and contains various values it needs to do it's job. 

This particular one is related to the [naming pattern for release branches ](https://internaldocs.unity.com/packageworks/wrench/api/RecipeEngine.Modules.Wrench.Settings.WrenchSettings/#branchnamingpattern)  So we can create the triggers for the jobs.